pkg/kotsclient/release.go

```diff
@@ -48,72 +30,58 @@ type KotsRelease struct {
 	Channels     []*KotsChannel `json:"channels"`
 }
 
-type GraphQLResponseUpdateKotsRelease struct {
-	Data *KotsReleaseUpdateData `json:"data,omitempty"`
-}
-
-type KotsReleaseUpdateData struct {
-	UpdateKotsRelease *UpdateKotsRelease `json:"updateKotsRelease"`
-}
-
-type UpdateKotsRelease struct {
-	ID     string `json:"id"`
-	Config string `json:"spec,omitempty"`
-}
-
-const createReleaseQuery = `
-mutation createKotsRelease($appId: ID!, $spec: String) {
-	createKotsRelease(appId: $appId, spec: $spec) {
-		sequence
+func (c *VendorV3Client) CreateRelease(appID string, multiyaml string) (*types.ReleaseInfo, error) {
+	gzipData := bytes.NewBuffer(nil)
+	gzipWriter := gzip.NewWriter(gzipData)
+	_, err := io.Copy(gzipWriter, strings.NewReader(multiyaml))
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to write gzip data")
 	}
-}`
 
-func (c *GraphQLClient) CreateRelease(appID string, multiyaml string) (*types.ReleaseInfo, error) {
-	response := GraphQLResponseKotsCreateRelease{}
+	if err := gzipWriter.Close(); err != nil {
```

Comment:
probably should close this regardless of io.Copy error

Reply:
eh.. it's an in-memory buffer, not a file.  updated.